doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -145,57 +107,61 @@ Bluetooth mesh
 
   * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
 
-Matter
-------
+nRF9160 samples
+---------------
 
-* Added:
+* :ref:`https_client` sample:
 
-  * Multi-image Device Firmware Upgrade over Bluetooth LE support for nRF5340 DK in lock and light bulb samples.
+  * Added a possibility to use TF-M and Zephyr Mbed TLS instead of using the offloaded TLS stack in modem.
 
-Zigbee
-------
+* :ref:`lwm2m_client` sample:
 
-* Added:
+  * Added support for Thingy:91.
+  * Added more LwM2M objects.
+  * LwM2M sensor objects now uses the actual sensors available to the Thingy:91. If the nRF9160 DK is used, it uses simulated sensors instead.
+  * Added possibility to poll sensors and notify the server if the measured changes are large enough.
 
-   * :ref:`Zigbee shell <zigbee_shell_sample>`.
-   * Added ZCL commands to :ref:`Zigbee Shell library <lib_zigbee_shell>`.
-   * Fixes and improvements in :ref:`Zigbee Shell library <lib_zigbee_shell>`.
-   * Fixed issue with cluster declaration in :ref:`Zigbee shell <zigbee_shell_sample>` and :ref:`Zigbee template <zigbee_template_sample>`.
-   * Improve logging in :ref:`ZBOSS OSIF <lib_zigbee_osif>` and :ref:`Zigbee Shell library <lib_zigbee_shell>`.
+* :ref:`multicell_location` sample:
```

Comment:
This is merged but I'm changing this with my PR because the changes applies to multicell_location library mainly and not all of these affect the sample.

My PR is https://github.com/nrfconnect/sdk-nrf/pull/5881